src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -830,6 +840,20 @@ impl Annotation {
                         }
                     },
                     //update runtume config
```

Comment:
```suggestion
                    //update runtime config
```

Reply:
Thanks for your review, already updated.